src/dvclive/live.py

```diff
@@ -345,6 +362,7 @@ def sync(self):
 
         self.make_report()
 
+        logger.warning(f"post_to_studio step{self.step}")
```

Comment:
Do we want to keep these or were they only for debugging? If we keep them, let's lower the logging level so they don't show by default and aren't considered warnings.

Reply:
They are only for debugging propose :)